src/app/lists/tree/tree-advanced-sample/services/data.service.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Please move the service under `./services`.

Reply:
@kdinev 
To be clear, these should be moved under `app/{featureModule}/services` folder (in this case, `app/lists/services`) as opposed to `app/services` (which is a separate feature module)?

Reply:
You can leave the service as is, but the data should be moved to the data folder.

Reply:
Thanks for clarifying! Data's now moved to the root (`app/data`) folder